source/npmhub.js

```diff
@@ -58,12 +58,15 @@ async function init() {
     ]),
   });
 
-  const buttonTarget = document.querySelector(['.BorderGrid-cell [href=\'#readme\']']).parentNode;
-  const frag = document.createDocumentFragment();
+  const position = document.querySelector(['.BorderGrid-cell [href*=\'report-content\']'])?.parentElement;
```

Comment:
The report button isn’t available everywhere. For example you can’t report your own repos